WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingHomeViewModel.kt

```diff
@@ -24,17 +29,18 @@ class PrepublishingHomeViewModel @Inject constructor(
     private val _onActionClicked = MutableLiveData<Event<ActionType>>()
     val onActionClicked: LiveData<Event<ActionType>> = _onActionClicked
 
-    fun start(editPostRepository: EditPostRepository) {
+    fun start(editPostRepository: EditPostRepository, site: SiteModel) {
         if (isStarted) return
         isStarted = true
 
-        setupHomeUiState(editPostRepository)
+        setupHomeUiState(editPostRepository, site)
     }
 
     // TODO remove hardcoded Public with live data from the EditPostRepository / user changes.
-    private fun setupHomeUiState(editPostRepository: EditPostRepository) {
+    private fun setupHomeUiState(editPostRepository: EditPostRepository, site: SiteModel) {
         val prepublishingHomeUiStateList = listOf(
-                PrepublishingHomeUiState(
+                HeaderUiState(UiStringText(site.name), site.iconUrl?.let { it } ?: run { "" }),
+                HomeUiState(
```

Comment:
Let me know if this logic is clear or it needs to be reworked for clarity. Basically, if the icon URL is `null` I am passing an empty string so that the `ImageManager` can fallback to a placeholder.

Reply:
I wouldn't mind this logic, but since we have a utility method just for this purpose, I'd consider using it - StringUtils.notNullStr(). Imo it's okay to use a static access for this purpose since we don't want to mock this type of utility method anyway.

Reply:
Oh nice! Never knew that existed 🙏 
7759cac